### PR TITLE
JeanetteKnipp-Kursdoku

### PR DESCRIPTION
**Pull request by Lellahahaha**
Awesome you want to make a PR  
please see [https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md](https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md) for further infos.

### CODE REVIEW

File: projects/DMA-Enrico-Castellani/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  - "Natalie Schreiber"
+authors-url: 
+  - "https://github.com/lellahahaha"
+  - "https://github.com/natascher"
+layout: project
+title: "Deconstructing Master Artists - Enrico Castellani"
+type: project
+---
+
+![](./assets/castellani.png)
```
Comment (fabianmoronzirfas) [inline projects/DMA-Enrico-Castellani/index.md]: No splash.png image? 
no subfolder called images?

File: projects/DMA-Enrico-Castellani/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+authors-url: 
+  - "https://github.com/lellahahaha"
+  - "https://github.com/natascher"
+layout: project
+title: "Deconstructing Master Artists - Enrico Castellani"
+type: project
+---
+
+![](./assets/castellani.png) 
+# Deconstructing Master Artists
```
Comment (fabianmoronzirfas) [inline projects/DMA-Enrico-Castellani/index.md]: The Headline h1 (`#`) will be added by the system form your title. See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax

File: projects/DMA-Enrico-Castellani/index.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+***Law of foreshortening***
+
+![](./assets/foreshortening.png) 
+
+If similar objects are being lined up and contracted towards one side, this appears optically as a crushed movement.
+
+------
+
+We tried to express the laws used by Castellani in code to create generative art. This is the result:
```
Comment (fabianmoronzirfas) [inline projects/DMA-Enrico-Castellani/index.md]: You are very short on your outcome…

File: projects/this-is-not-a-leaf/index.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+This way you get a wider range of possible correct outcomes while increasing the space for creativity.
+
+----------
+
+### The Revised Second Algorithm
+
+
+The revised algorithm now looks like this:
+
+*Draw ovals that thin to the tails. crisscross, in different sizes, almost touching. Fill some of them with lines or dots and leave others empty.*
```
Comment (fabianmoronzirfas) [inline projects/this-is-not-a-leaf/index.md]: maybe better as quote:   

> Draw ovals that thin to the tails. crisscross, in different sizes, almost touching. Fill some of them with lines or dots and leave others empty.

done like this  

    > Draw ovals that thin to the tails. crisscross, in different sizes, almost touching. Fill some of them with lines or dots and leave others empty.

File: projects/this-is-not-a-leaf/index.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+### Changing Constants
+
+How about changing the leaf-like shapes to clouds, hearts and stars?
+
+![](./assets/ergebnis11.png)
+![](./assets/ergebnis12.png)
+![](./assets/ergebnis13.png)
+
+As you can see, with this new algorithm it still works out. All correct results! But a little corny now. So let’s stay with the pseudo-leafs and change the canvas! No regular sheets of paper allowed anymore. But instead - wrapping paper, an egg, even skin.
```
Comment (fabianmoronzirfas) [inline projects/this-is-not-a-leaf/index.md]: Please write from a bit more objective perspective. No slang, no "let's try…" Write as it where an analysis

File: projects/this-is-not-a-leaf/index.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+Nice try, but unfortunately it does not look that professional after all. So what about painting on shirts, envelopes, glass, porcellain?
+
+![](./assets/shirt2.png)
+![](./assets/brief1.png)
+![](./assets/glass.png)
+![](./assets/teller1.png)
+![](./assets/teller2.png)
+
+Yassss!
```
Comment (fabianmoronzirfas) [inline projects/this-is-not-a-leaf/index.md]: please… This is not twitter or facebook.

**Comment by fabianmoronzirfas** [comment]
@Lellahahaha are you done? can I review it again?

File: projects/DMA-Enrico-Castellani/index.md (hunk 2 above)
Comment (Lellahahaha) [inline projects/DMA-Enrico-Castellani/index.md]: tried it, doesn't work for me.

File: projects/DMA-Enrico-Castellani/index.md (hunk 3 above)
Comment (Lellahahaha) [inline projects/DMA-Enrico-Castellani/index.md]: changed it.

File: projects/this-is-not-a-leaf/index.md (hunk 4 above)
Comment (Lellahahaha) [inline projects/this-is-not-a-leaf/index.md]: done.

File: projects/this-is-not-a-leaf/index.md (hunk 5 above)
Comment (Lellahahaha) [inline projects/this-is-not-a-leaf/index.md]: changed

**Comment by fabianmoronzirfas** [comment]
There is still the question of the image rights open.
Also you've added a folder on the root level of the project called images:  

https://github.com/Lellahahaha/gestalten-in-code/tree/e330320be8063031a5e5384945b57ca58a216a5e/images 

Please remove it. All your images should be in your project/sketches folders

File: projects/DMA-Enrico-Castellani/index.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+
+![](./assets/images/castellanicode.png)
+
+In concliusion, even though Castellani's work is "only" higher and lower placed nails on an evenly colored surface it's very hard to imitate using code. The nails use light and give shadow to appear as dots, lines and whole forms in a three dimensional space which is, for p5.js-beginners, a great task to authentically simulate - only having the knowledge to use basic shapes like circles and connected lines and consistent colors. Maybe gradients would have been more efffective in showing what light and shadow does to darker spots.
```
Comment (fabianmoronzirfas) [inline projects/DMA-Enrico-Castellani/index.md]: > concliusion
conclusion

**Comment by fabianmoronzirfas** [comment]
So? Any development on the images?

**Comment by Lellahahaha** [comment]
Almost.
Unfortunately, I couldn't find any license free fitting high resolution images so far (although I have been searching for them on at least three different Pages.)
Will try it again now - if I still can't find any I'll just remove the Images from the document. Will update it till 7PM today.

**Comment by fabianmoronzirfas** [comment]
Hm. Sad. If you find anything low res use that

**Comment by Lellahahaha** [comment]
Alright!

**Comment by Lellahahaha** [comment]
Okay, so:

Changed the splash and thumb Image into one that I don't Need a license for. Also, I found one license free Picture as an example for Castellani's work. But except for that I really could not find any free Images. Neither of Castellani himself, nor of his work or the working process. So I removed all the Images I do not have a license for from the document and the Image Folder. But: what do I do about the examples Natalie and I made in Illustrator by only covering existing Images of castellani's work with light lines? Do I have to delete them as well because we don't have the license for the original Pictures? Or should I just blur them a Little? Or can I leave them as they are?

**Comment by fabianmoronzirfas** [comment]
> Do I have to delete them as well because we don't have the license for the original Pictures? Or should I just blur them a Little? Or can I leave them as they are?

Let's keep it that way for now.

**Comment by fabianmoronzirfas** [comment]
merged and live https://interface.fh-potsdam.de/gestalten-in-code/projects/DMA-Enrico-Castellani/ && https://interface.fh-potsdam.de/gestalten-in-code/projects/this-is-not-a-leaf/

**Pull request merged**